hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestHRegion.java

```diff
@@ -6281,6 +6288,44 @@ public void testBulkLoadReplicationEnabled() throws IOException {
         getCoprocessors().contains(ReplicationObserver.class.getSimpleName()));
   }
 
+  @Test 
```

Comment:
whitespace:end of line

Reply:
leaving feadback on the PR lines instead of making someone dig into the reporting worked! oh man that's great.

Reply:
It's only whitespace I think... It will be good if we could also make findbugs and checkstyle work like this...